ci-operator/jobs/openshift/cluster-api/openshift-cluster-api-master-presubmits.yaml

```diff
@@ -4,6 +4,7 @@ presubmits:
     always_run: true
     branches:
     - master
+    - release-4
```

Comment:
Not sure if we've already created the branch on the upstream repo, but we might consider using a scheme similar to the one the master team mentioned, e.g.: `openshift-4.0-k8s-1.12.0` or something.